explorer/indexer/app/testdata/default_explorer_indexer.toml

```diff
@@ -10,7 +10,7 @@ version = "unknown"
 #######################################################################
 
 # URL at which the postgres database can be reached.
-dburl = "postgres://omni:password@db:5432/omni_db"
+dburl = "postgres://omni:password@explorer_db:5432/omni_db"
```

Comment:
same as above

Reply:
Actually, probably these two should be an empty string and should be passed by the deployment config/docker config.